patrick/advanced-javascript/src/arrays.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-unused-vars, max-len */
```

Comment:
A bit extreme on the spacing here.

Reply:
I never did figure out why this files showed up on GitHub with such ridiculous spacing... I think I was using tabs

...but then I tried pizza on pineapple... and everything was alright again.